src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Authorization.EmployerUserRoles.Options;
+using SFA.DAS.Authorization.Mvc.Attributes;
+using SFA.DAS.Employer.Shared.UI;
+using SFA.DAS.Employer.Shared.UI.Attributes;
+using SFA.DAS.EmployerCommitmentsV2.Features;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice;
+using SFA.DAS.EmployerCommitmentsV2.Web.RouteValues;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Controllers
+{
+    [Route("{accountHashedId}/apprentices")]
+    [SetNavigationSection(NavigationSection.ApprenticesHome)]
+    [DasAuthorize(EmployerUserRole.OwnerOrTransactor)]
+    public class ApprenticeController : Controller
+    {
+        [Route("", Name = RouteNames.ManageApprentices)]
+        [DasAuthorize(EmployerFeature.ManageApprenticesV2)]
+        public IActionResult Index(string accountHashedId)
```

Comment:
Following the pattern, i think this will need to be IndexRequest with accountHashedId within it

Reply:
done in MF-431